FParsecCS/Text.cs

```diff
@@ -363,11 +363,12 @@ public static int CountTextElements(string str)
     return new System.Globalization.StringInfo(str).LengthInTextElements;
 }
 
-// Apparently System.Char.Is(High|Low)Surrogate is not safe for consumption by Silverlight developers
-
-public static bool IsSurrogate(char ch)     { return (ch & 0xF800) == 0xD800; }
```

Comment:
The definitions here are slightly simpler than the system ones, but that shouldn't make a difference on modern processors, and hopefully not for the JIT inliner/optimizer either.